docusaurus.config.js

```diff
@@ -59,25 +59,11 @@ const config = {
     /** @type {import('@docusaurus/preset-classic').ThemeConfig} */
     ({
       navbar: {
-        title: "Home",
+        title: "Developer",
         logo: {
           alt: "EMnify Logo",
```

Comment:
It'd be great to add more descriptive alt text here:

```suggestion
          alt: "emnify Documentation – Home",
```

(could also be `Developer Documentation` or some other variation, up to you)

Reference: [Deque](https://www.deque.com/blog/great-alt-text-introduction/#:~:text=At%20a%20minimum%2C%20the%20alt,%E2%80%9CUser%20Testing%20%E2%80%93%20Home%E2%80%9D.)